MarkupsToModel/MRML/vtkMRMLMarkupsToModelNode.cxx

```diff
@@ -159,6 +164,19 @@ void vtkMRMLMarkupsToModelNode::ReadXMLAttributes( const char** atts )
       }
       SetInterpolationType(interpolationType);
     }
+    else if ( ! strcmp( attName, "PointParameterType" ) )
+    {
+      int pointParameterType = 0;
+      std::stringstream nameString;
+      nameString << attValue;
+      nameString >> pointParameterType;
```

Comment:
I'm a little bit confused - in what way am I treating this variable differently from InterpolationType? The code looks pretty much the same to me...